packages/Webkul/Shop/src/Resources/views/customers/forgot-password.blade.php

```diff
@@ -106,13 +106,13 @@ class="primary-button block w-full max-w-[374px] m-0 ml-[0px] mx-auto px-[43px]
         </div>
 
         <p class="mt-[30px] mb-[15px] text-[#6E6E6E] text-xs text-center">
-            @lang('shop::app.customers.forgot-password.footer')
+            @lang('shop::app.customers.forgot-password.footer',['current_year'=> date('Y') ])
```

Comment:
Please add one space after the (, ) symbols in all blade files.

Reply:
@amit-webkul its done